Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -713,6 +716,41 @@ public boolean isEcoDisabled() {
         return economyDisabled;
     }
 
+    private boolean buySignUpdatedToWorth = false;
+
+    private boolean _getBuySignUpdatedToWorth() {
+      return config.getBoolean("economy-update-buy-sign-to-worth", false);
+    }
+
+    @Override
+    public boolean isBuySignUpdatedToWorth() {
+      return buySignUpdatedToWorth;
+    }
+
+    private boolean sellSignUpdatedToWorth = false;
+
+    private boolean _getSellSignUpdatedToWorth() {
+      return config.getBoolean("economy-update-sell-sign-to-worth", false);
+    }
+
+    @Override
+    public boolean isSellSignUpdatedToWorth() {
+      return sellSignUpdatedToWorth;
+    }
```

Comment:
These config options do not require a helper method, you can inline the config.getBoolean call
```suggestion
    @Override
    public boolean isBuySignUpdatedToWorth() {
      return config.getBoolean("economy-update-buy-sign-to-worth", false);
    }

    @Override
    public boolean isSellSignUpdatedToWorth() {
      return config.getBoolean("economy-update-sell-sign-to-worth", false);
    }
```

Reply:
I was wondering where those helper functions were for :)